src/DataReader/json_reader.rs

```diff
@@ -49,7 +49,7 @@ pub fn read_json_component(project_path: &str, component_name: &str) -> componen
 //Description:uses the filename to open the file and then reads the file.
 //Output: Result type, if more info about this type is need please go to: https://doc.rust-lang.org/std/result/
 pub fn read_json<T: DeserializeOwned>(filename: &str) -> serde_json::Result<T> {
-    let mut file = File::open(filename).unwrap();
+    let mut file = File::open(filename).expect(&format!("Could not find file {filename}"));
```

Comment:
Should this be part of this pull request?
(also a clippy complaint on this)

Reply:
Yes, Sebastian added it for just for a better error msg